.github/workflows/test-upstream.yml

```diff
@@ -62,7 +27,7 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-v1-jdk${{ matrix.java }}-${{ hashFiles('**/pom.xml') }}
+          key: ${{ runner.os }}-maven-v1-${{ hashFiles('**/pom.xml') }}
```

Comment:
We can probably delete this entire caching step now if we aren't caching any build output

Reply:
Do we still need to include a build step above for this workflow to run?

Reply:
Yeah, we would need to remove the `needs: build` line for this job since the `build` job has been removed, although based on conversation [here](https://github.com/dask-contrib/dask-sql/pull/471/files/9936c92930019e11e0e5e5dc7d3831c6ae1ca9e4#r849618927) I think we might prefer keeping the `build` jobs in and just altering them to build / cache the Rust code instead